### ISSUE TITLE
The docstrings for `test_ownClass` has typos

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6022]
|<img alt="gehrhorn's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| gehrhorn reported|
|-|-|
|Trac ID|trac#6022|
|Type|enhancement|
|Created|2012-09-26 01:00:22Z|

The docstring for test_ownClass in twisted/python/test/test_reflectpy3.py reads

"""
If x is **and** instance of **Base}** and Base defines a method named method, L{accumulateMethods} adds an item to the given dictionary with C{"method"} as the key and a bound method object for Base.method value.
"""

I've attached a patch that fixes the typos.

Attachments:

* [patch.txt](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a58/a58dcb7f4d5593428bac0dd85c941f216c6fcd34/3f09e3af32b63927e59020b0a1e33673b70c36c8.txt) (629 bytes) - added by gehrhorn on 2012-09-26 01:01:22Z - Patch to fix typos in docstring for test_ownClass()

<details><summary>Searchable metadata</summary>

```
trac-id__6022 6022
type__enhancement enhancement
reporter__gehrhorn gehrhorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1348621222000000 1348621222000000
changetime__1348936240000000 1348936240000000
version__None None
owner__therve therve

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve set owner to @therve|
|-|-|

Looks good, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set status to `closed`|
|-|-|

(In [35845]) Fix a docstring in twisted/python/test/test_reflectpy3.py, plus some other
formatting cleanups.

Author: gehrhorn
Reviewer: therve
Fixes: [#6022](https://github.com/twisted/twisted/issues/6022)
